### DISCUSSION TITLE
ghippo/user-guide/personal-center/security-setting/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3426] [Announcements]
# ghippo/user-guide/personal-center/security-setting/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/ghippo/user-guide/personal-center/security-setting/

**Comment by Fan-Lin** [comment]
不知道为什么点击全局管理那一步，和后文衔接不上，同时缺乏安全设置具体介绍。

**Comment by windsonsea** [comment]
Fixed in PR #5287
